### PR TITLE
add initial schemas for Telco Guide 1.0

### PR DESCRIPTION
**Pull request by ammend**
It took a long time to calibrate json schemas for Telco guide 1.0. The underlying logic can be see in [link](https://github.com/ammend/Telco-WG/tree/main/schemas). This pull request add a json schema for Telco guide 1.0 and give comparison between spdx 2.2 and 2.3 and developed a python script to do this. Thanks for your review!

### CODE REVIEW

File: schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Schemas of OpenChain Telco Guide
+This directory is an implementation of json schemas of Telco Guide. The file [openchain-telco-guide-v1.0-schema.json](openchain-telco-guide-v1.0-schema.json) is just the json schema file you can take to your project. 
+
+As further described below, a technology viewpoint is shown below how to generate json schema file from scratch if you wanna a clear comprehension of underlying logic.
+
+## SPDX 2.2 and SPDX 2.3
+In the specification of OpenChain Telco Guide, we say 'An OpenChain Telco SBOM Guide compatible document SHALL adhere to the version 2.2 of the SPDX Data Format as standardized in ISO/IEC 5962:2021, or to the version 2.3 of the standard'. By 05/08/2024, v2.2, v2.2.1, v2.2.2 and v2.3 have been released in github website. We choose v2.2.2 and v2.3 as origin json schema implementation.
```
Comment (winterrocks) [inline schemas/README.md:7]: I would avoid date formats like "05/08/2024". My personal preference is ISO 8061 yyyy-mm-dd which tends to have the least opportunities for misinterpretations. Wiki: https://en.wikipedia.org/wiki/ISO_8601
When using "05/08/2024" it is difficult to say if we are talking about May 8th or August 5th.

Comment (ammend) [reply schemas/README.md:7]: Thanks for your review. I will modify it later.

**Comment by vargenau** [comment]
Hello,

Thank you for your contribution.

I propose that we review it during our next monthly meeting on 5th of September. Will you be able to join in one of the two sessions?

Best regards,

Marc-Etienne

--
Marc-Etienne Vargenau ***@***.******@***.***>
Nokia, 12, rue Jean-Bart, 91300 Massy, FRANCE
Mobile: +33 6 24 49 78 68<tel:+33624497868>
Senior Specialist Open Source
Planned absence: none


De : ammend ***@***.***>
Date : lundi, 5 août 2024 à 07:35
À : OpenChain-Project/Telco-WG ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [OpenChain-Project/Telco-WG] add initial schemas for Telco Guide 1.0 (PR #91)

It took a long time to calibrate json schemas for Telco guide 1.0. The underlying logic can be see in link<https://github.com/ammend/Telco-WG/tree/main/schemas>. This pull request add a json schema for Telco guide 1.0 and give comparison between spdx 2.2 and 2.3 and developed a python script to do this. Thanks for your review!

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/OpenChain-Project/Telco-WG/pull/91

Commit Summary

  *   d96eec7<https://github.com/OpenChain-Project/Telco-WG/pull/91/commits/d96eec7c6ecf512a4f1c9ab137427ee713cd7df8> add initial schemas for Telco Guide 1.0
  *   d4319c0<https://github.com/OpenChain-Project/Telco-WG/pull/91/commits/d4319c0cc13922873b113143db33116a83cb7536> Merge branch 'OpenChain-Project:main' into main

File Changes

(7 files<https://github.com/OpenChain-Project/Telco-WG/pull/91/files>)

  *   A schemas/README.md<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-b0566199e40770510d3d5ab842ca8936fbbfc552998e25174060f97e45a4a234> (98)
  *   A schemas/internal/json_schema_compare.py<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-47cf2bae546164cdebdce4b2356c61a6e9b735b265d6c035a5a5f77458552702> (129)
  *   A schemas/internal/spdx-v2.2.2-fix-schema.json<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-9c2dcdb27cb8b9c89be7045db4ff90e3119488dcaae6dfca09505f3446b1e5bb> (893)
  *   A schemas/internal/spdx-v2.2.2-origin-schema.json<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-9a59c84580c80a592739110b237499ea9d3bfe383612a5f087f44c7f510dbd29> (905)
  *   A schemas/internal/spdx-v2.3-fix-schema.json<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-af18bba759ea4499ac1bf04c7a7f417ea1bc96536e71b862aa65d06eb7ed25ff> (915)
  *   A schemas/internal/spdx-v2.3-origin-schema.json<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-54765ab2b385476670e54753914b8adc8230ad0d7b4953cc3fd4d81c581a5fed> (961)
  *   A schemas/openchain-telco-guide-v1.0-schema.json<https://github.com/OpenChain-Project/Telco-WG/pull/91/files#diff-c95c11bdd91ac1e43e1ac165a7dd262f06a3cc89a8762d7549a3e3b03a27d194> (924)

Patch Links:

  *   https://github.com/OpenChain-Project/Telco-WG/pull/91.patch
  *   https://github.com/OpenChain-Project/Telco-WG/pull/91.diff

**Comment by ammend** [comment]
Thanks for your review! I will join in the meeting in 2024-09-25 – 08:00 UTC.

**Pull request merged**